api/taskgroup.go

```diff
@@ -176,6 +176,7 @@ func (h TaskGroupHandler) Update(ctx *gin.Context) {
 			})
 		return
 	}
+	db = db.Omit("Bucket")
```

Comment:
The read-only permission removed from the model so we need to prevent updating the bucket ref in the API.